### PR TITLE
google-app-engine-go-sdk: init at 1.9.53

### PR DESCRIPTION
**Pull request by lufia**
###### Motivation for this change

AppEngine SDK for Go is tools for developing, deploying, and managing apps in Google App Engine.

###### Things done

- [x] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS
- Built on platform(s)
   - [x] NixOS
   - [x] macOS
   - [ ] Linux
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/development/tools/google-app-engine-go-sdk/default.nix [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+      ln -s "$out/share/go_appengine/$i" "$out/bin/$i"
+    done
+  '';
+
+  meta = with stdenv.lib; {
+    description = "Google App Engine SDK for Go";
+    version = version;
+    homepage = "https://cloud.google.com/appengine/docs/go/";
+    license = licenses.asl20;
+    platforms = with platforms; linux ++ darwin;
```
Comment (Mic92) [inline pkgs/development/tools/google-app-engine-go-sdk/default.nix]: Do you want to maintain this package?

Comment (lufia) [inline pkgs/development/tools/google-app-engine-go-sdk/default.nix]: I'm so happy to maintain this package.
I will push new commits that add myself to maintainer list.

**Comment by lufia** [comment]
hmm.. I installed this package on macOS after `nix-channel --update`.
install is succeed but `goapp` fails.

```console
/Users/lufia/.nix-profile/bin/goapp: line 7: Convenience wrapper for starting a Go tool in the App Engine SDK.: command not found
ssh: Could not resolve hostname argparse: nodename nor servname provided, or not known
import: EOF
ssh: Could not resolve hostname os: nodename nor servname provided, or not known
import: EOF
```

`goapp`'s code that raises this errors is:

```python
#!/nix/store/37sk1sjzmky15nkxvyrw97sizx7fhlf2-python-2.7.13-env/bin/python2.7
#
# Copyright 2011 Google Inc. All rights reserved.
# Use of this source code is governed by the Apache 2.0
# license that can be found in the LICENSE file.

"""Convenience wrapper for starting a Go tool in the App Engine SDK."""

import argparse
import os
import sys
```

and _/nix/store/37sk1sjzmky15nkxvyrw97sizx7fhlf2-python-2.7.13-env/bin/python2.7_:

```bash
#! /nix/store/d0vj5xfr85716dz2k8xb6fhygxgllk3b-bash-4.4-p12/bin/bash -e
export PYTHONHOME="/nix/store/37sk1sjzmky15nkxvyrw97sizx7fhlf2-python-2.7.13-env"
exec "/nix/store/95ssabrarnq5w597y51c66zmvkd44xh0-python-2.7.13/bin/python2.7"  "${extraFlagsArray[@]}" "$@"
```

Perhaps, macOS's shebang requires to be binary.

```console
$ cat wrap
#!/bin/bash -e

exec /usr/bin/python "$@"
$ cat hello
#!./wrap

if __name__ == '__main__':
	print "hello"
$ ./wrap hello
hello
$ ./hello
./hello: line 5: syntax error: unexpected end of file
```

I think this error is issue of python27.withPackages on macOS.
What do you think?

**Comment by Mic92** [comment]
@FRidh have you have heard of that issue before?

**Comment by FRidh** [comment]
@lufia could very well be that we cannot use `python.withPackages` because of https://github.com/NixOS/nixpkgs/issues/11133.

**Pull request merged**